app/models/dataset.rb

```diff
@@ -34,33 +54,9 @@ def non_timeseries_datafiles
     datafiles.select(&:non_timeseries?)
   end
 
-  class << self
-    def from_json(raw_json)
-      d = Dataset.new(raw_json.merge(raw_json['_source']))
-      d.json = raw_json
```

Comment:
@laurentqro - I notice that the 'json' property is no longer there. Might be worth checking with @govuklaurence if this is ok. I've never used it but there may be a reason why it is there.

Reply:
It's most likely way outdated code that was left over